app/routers/getjoke.py

```diff
@@ -0,0 +1,10 @@
+import requests
+from fastapi import APIRouter
+
+
+router = APIRouter()
+
+
+@router.get("/getjoke")
```

Comment:
Just `/joke`, the GET is the HTTP Verb itself

Reply:
done.